shared/actions/chat/messages.js

```diff
@@ -161,10 +161,15 @@ function* editMessage(action: Constants.EditMessage): SagaGenerator<any, any> {
   // Not editing anymore
   yield put(Creators.showEditor(null))
 
+  // if message post-edit is the same as message pre-edit, skip call and marking message as 'EDITED'
+  const [prevMessageText, newMessageText] = [message.message.stringValue(), action.payload.text.stringValue()]
```

Comment:
why build this array and then just undo it? vs just 2 consts

Reply:
good point